docs/content/envoy/support.end.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Envoy Support"
+date: 2024-12-10T13:53:33-05:00
+lastmod: 2024-12-10T13:53:33-05:00
+description: "Envoy Support"
+weight: 100
+---
+
+## How can we help?
+
+
+|                        | Open Source            | One-Time Setup         | Managed Service |
```

Comment:
This table comes from the Envoy repo's README. If additional information should be included, let me know.